CHANGELOG.md

```diff
@@ -27,7 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Internal
 
-# Past Releases
```

Comment:
I've removed the `# Past Releases` section of the changelog to make it easier to stamp new versions. I can probably figure out a way to keep it but it seemed a bit unusual anyways.